charts/helm-controller-operator/templates/deployment.yaml

```
@@ -0,0 +1,164 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: helm-controller
+  namespace: {{ .Values.namespace }}
+spec:
+  progressDeadlineSeconds: {{ .Values.progressDeadlineSeconds }}
+  replicas: {{ .Values.replicaCount }}
+  revisionHistoryLimit: {{ .Values.namespacerevisionHistoryLimits }}
+  selector:
+    matchLabels:
+      app: helm-controller
+  {{- with .Values.manager }}
+  strategy:
+    {{- toYaml .helmControllerStrategy | nindent 4 }}
+  template:
+    metadata:
+      annotations:
+        prometheus.io/port: "8080"
+        prometheus.io/scrape: "true"
+      labels:
+        app: helm-controller
```

Cf : use the function from the tpl generated by helm
```suggestion
      labels:
        {{- include "helm-controller-operator.selectorLabels" . | nindent 8 }}
```